dist/api_data.py

```diff
@@ -1528,7 +1528,12 @@ def __ge__(self, other):
         Config('release_evict', 'false', r'''
             Configure the cursor to evict the page positioned on when the
             reset API is used''',
-            type='boolean')
+            type='boolean'),
+        Config('dump_version', 'false', r'''
+            open a version cursor, which is a debug cursor on a table that
+            enables iteration through all updates/versions of a key inside
+            the table.''',
```

Comment:
I think this makes it sounds like you are looking at different keys.  Perhaps: `enables iteration through the history of values for a given key`.